Podfile

```diff
@@ -1,14 +1,14 @@
 source 'https://github.com/CocoaPods/Specs'
 
-platform :ios, '8.0'
+platform :ios, '9.0'
 
 use_frameworks!
 inhibit_all_warnings!
 
 def commonpods
   pod 'IP-UIKit-Wisdom'
-    pod 'RxSwift', '~> 4.0'
-    pod 'RxCocoa', '~> 4.0'
+    pod 'RxSwift', '~> 4.3.0'
```

Comment:
Do you want this (and all of the others) to be `~> 4.3.0`, or `~> 4.3`?

Reply:
For posterity capturing an offline convo: Do the same change for the podspec.